### PR TITLE
Fix headers

### PR DESCRIPTION
**Pull request by b-scholz**
Tidied up header fiels

### CODE REVIEW

File: src/StringPool.h [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2017 The Souffle Developers. All Rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file SringPool.h
```
Comment (mmcgr) [inline src/StringPool.h]: Missing 't': StringPool

**Comment by souffle-lang** [comment]
[64% (-0.03%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/113/)

**Comment by souffle-lang** [comment]
[64% (0.0%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/114/)

**Comment by souffle-lang** [comment]
[64% (-0.04%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/115/)

**Comment by souffle-lang** [comment]
[64% (-0.01%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/116/)

**Comment by souffle-lang** [comment]
[64% (-0.01%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/117/)

**Comment by souffle-lang** [comment]
[64% (+0.01%) vs master 64%](http://plang1.it.usyd.edu.au/jenkins/job/PR-RHEL7-gcc7-Coverage/118/)

**Pull request merged**